### PR TITLE
Add function for computing square root

### PR DESCRIPTION
**Pull request by dexter2206**
Add square function that computes x^2.

### CODE REVIEW

File: functions.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+def square(x):
```
Comment (dexter2206) [inline functions.py:3]: This function computes square.

Comment (rmarjasz) [reply functions.py:3]: Acknowledged

Comment (rmarjasz) [reply functions.py:3]: Doc string is missing.

**Review by rmarjasz — CHANGES_REQUESTED** [review]

File: functions.py (hunk 1 above)
Comment (dexter2206) [reply functions.py:3]: Docstrings fixed in f8b61ae

**Review by rmarjasz — APPROVED** [review]

**Pull request merged**